tests/msc2716_test.go

```diff
@@ -99,7 +99,7 @@ func TestImportHistoricalMessages(t *testing.T) {
 			//
```

Comment:
Reviewers note: A lot of the diff has been split out into separate PRs:

 - https://github.com/matrix-org/complement/pull/250
 - https://github.com/matrix-org/complement/pull/248
 - https://github.com/matrix-org/complement/pull/251
 - https://github.com/matrix-org/complement/pull/249

Reply:
All of these are merged in now 🚀